File: testers/avro/scala/generated-and-checked-in/com/example/events/consumer/DynamicValueHandler.scala
package com.example.events.consumer

import com.example.events.DynamicValue
import com.example.events.header.StandardHeaders

/** Handler interface for dynamic-value topic events */
trait DynamicValueHandler {
  /** Handle a message from the topic */
  def handle(
    key: String,
    value: DynamicValue,
    headers: StandardHeaders
  ): Unit
}

File: testers/avro/java-async/generated-and-checked-in/com/example/events/consumer/DynamicValueHandler.java
package com.example.events.consumer;

import com.example.events.DynamicValue;
import com.example.events.header.StandardHeaders;
import java.util.concurrent.CompletableFuture;

/** Handler interface for dynamic-value topic events */
public interface DynamicValueHandler {
  /** Handle a message from the topic */
  CompletableFuture<Void> handle(String key, DynamicValue value, StandardHeaders headers);
}
